resolwe/flow/serializers/relation.py

```diff
@@ -4,29 +4,27 @@
 from rest_framework import serializers
 
 from resolwe.flow.models.collection import Collection
-from resolwe.flow.models.entity import PositionInRelation, Relation, RelationType
+from resolwe.flow.models.entity import RelationPartition, Relation, RelationType
 from resolwe.rest.fields import ProjectableJSONField
 from resolwe.rest.serializers import SelectiveFieldMixin
 
 from .base import ResolweBaseSerializer
 
 
-class PositionInRelationSerializer(SelectiveFieldMixin, serializers.ModelSerializer):
-    """Serializer for PositionInRelation objects."""
-
-    position = ProjectableJSONField(allow_null=True, required=False)
+class RelationPartitionSerializer(SelectiveFieldMixin, serializers.ModelSerializer):
+    """Serializer for RelationPartition objects."""
 
     class Meta:
-        """PositionInRelationSerializer Meta options."""
+        """RelationPartitionSerializer Meta options."""
 
-        model = PositionInRelation
-        fields = ('entity', 'position')
+        model = RelationPartition
+        fields = ('id', 'entity', 'position', 'label')
 
 
 class RelationSerializer(ResolweBaseSerializer):
     """Serializer for Relation objects."""
 
-    entities = PositionInRelationSerializer(source='positioninrelation_set', many=True)
+    entities = RelationPartitionSerializer(source='relationpartition_set', many=True)
```

Comment:
It's a bit misleading to call this `entities`, since this is not a list of `entities` but of RelationPartition objects which include the entity, label and position.

Reply:
Don't ask me how to call this better :) Maybe @jberci can help ;)